src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/ccd/callback/PreSubmitCallbackResponse.java

```diff
@@ -13,8 +15,9 @@
     private final T data;
     private final Set<String> errors = new LinkedHashSet<>();
 
+    @JsonCreator
```

Comment:
Just wondering about these annotations -- are they doing something for us that wasn't handled previously?

Reply:
It would appear nothing, I think I was using class in a way that did need it and then I reverted